loop.h

```diff
@@ -28,5 +28,7 @@
 int mount_bind(char *src, char *dest);
 int mount_loop(char *src, char *dest, char *fstype, int *loop_fd, int *file_fd);
 int unmount_loop(char *dest, int loop_fd, int file_fd);
+int get_free_loop(char *devname);
```

Comment:
I guess we are not using these out of loop.c in the end?

Reply:
moved the implementation to script. No changes to export it